rotkehlchen/tests/api/test_aave.py

```diff
@@ -25,78 +30,138 @@
     wait_for_async_task,
 )
 from rotkehlchen.tests.utils.rotkehlchen import setup_balances
-from rotkehlchen.types import ChecksumEvmAddress, deserialize_evm_tx_hash
+from rotkehlchen.types import (
+    ChainID,
+    ChecksumEvmAddress,
+    EvmTokenKind,
+    Price,
+    deserialize_evm_tx_hash,
+)
 
-AAVE_BALANCESV1_TEST_ACC = '0xC2cB1040220768554cf699b0d863A3cd4324ce32'
-AAVE_BALANCESV2_TEST_ACC = '0x8Fe178db26ebA2eEdb22575265bf10A63c395a3d'
+if TYPE_CHECKING:
+    from collections.abc import Sequence
 
+AAVE_BALANCESV1_TEST_ACC: Final = '0xC2cB1040220768554cf699b0d863A3cd4324ce32'
+AAVE_BALANCESV2_TEST_ACC: Final = '0xf2d02719309c8d5F22498c2d432eDc8eB8683d83'
+AAVE_BALANCESV3_TEST_ACC: Final = '0x18FD4323ea221cD05Fb275256C8f3A0740D0Aaa2'
 
-@pytest.mark.parametrize('ethereum_accounts', [[AAVE_BALANCESV1_TEST_ACC, AAVE_BALANCESV2_TEST_ACC]])  # noqa: E501
-@pytest.mark.parametrize('ethereum_modules', [['aave']])
-def test_query_aave_balances(
-        rotkehlchen_api_server: APIServer,
-        ethereum_accounts: list[ChecksumEvmAddress] | None,
-) -> None:
-    """Check querying the aave balances endpoint works. Uses real data.
 
-    TODO: Here we should use a test account for which we will know what balances
-    it has and we never modify
-    """
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [[
+    AAVE_BALANCESV1_TEST_ACC, AAVE_BALANCESV2_TEST_ACC, AAVE_BALANCESV3_TEST_ACC,
+]])
+@pytest.mark.parametrize('ethereum_modules', [['aave']])
+def test_query_aave_balances(rotkehlchen_api_server: APIServer) -> None:
+    """Check querying the aave balances endpoint works. Uses real data for v1/v2."""
```

Comment:
What about v3? It's mocked? Why? Why not find an online position since it's a deployed and recent version and mock it in another test? At the least you should write it's mocked here in the docstring.

Reply:
I avoided that for v3 because for it to work it needs the token balances and the underlying token mappings in the db.